client/src/views/TrackViewer/Viewer.vue

```diff
@@ -131,7 +136,18 @@ export default defineComponent({
 
     const location = computed(() => store.state.location);
 
-    function removeTrack(trackId: TrackId) {
+    async function removeTrack(trackId: TrackId) {
+      if (selectedTrackId === null) {
+        return;
+      }
```

Comment:
I should be able to remove a track without having any selected track.

As far as I can tell, this conditional serves no purpose and can be removed.

Reply:
Really it should be if the referenced `trackId` is null.  The idea was to prevent the user from hitting delete while not having anything selected.  It would display the prompt.  I forgot about the fact that the TrackList has it's own method of deletion of objects.